.github/CONTRIBUTING.md

```diff
@@ -38,7 +38,7 @@ The HTML will then be generated in `docs/api/_build/html`.
 
 
 ## Tests
-When you create a pull request with changes, [Travis CI](https://travis-ci.org/) will run automatic tests.
+When you create a pull request with changes, [Travis CI](https://travis-ci.com/) will run automatic tests.
```

Comment:
Should we specify Github Actions here?

Reply:
We havent got an implementation for GitHub Actions for the main `tools` repo yet but worth adding in when we do 👍